src/lib/viewers/controls/media/MediaSettings.tsx

```diff
@@ -1,41 +1,34 @@
 import React from 'react';
 import noop from 'lodash/noop';
-import getLanguageName from '../../../lang';
-import MediaSettingsMenuAudioTracks, { AudioTrack, Props as AudioTracksProps } from './MediaSettingsAudioTracks';
+import MediaSettingsMenuAudioTracks, {
+    addLabels,
+    Props as AudioTracksProps,
+    QUALITY_LABEL_MAP,
```

Comment:
Should this live in the quality menu, instead? Can/should it be a function rather than the actual constant?

Reply:
LOL you're right, good catch